apps/web/components/AppListCard.tsx

```diff
@@ -60,7 +60,7 @@ export default function AppListCard(props: AppListCardProps) {
         timeoutRef.current = null;
       }
     };
-  }, []);
+  }, [highlight, router, shouldHighlight]);
```

Comment:
This is very risky, we can't merge this - I pushed a suggestion to https://github.com/calcom/cal.com/pull/8621 which @keithwillcode will evaluate, after which we can re-run the fixer and hopefully get the desired fixups without breaking anything.

Reply:
@emrysal My thoughts [here](https://github.com/calcom/cal.com/pull/8621/files#r1192590344).